server/datastore/mysql/migrations/tables/20230926082152_AddWindowsMDMTables_to_be_deleted.go

```diff
@@ -11,7 +11,7 @@ func init() {
 
 func Up_20230926082152(tx *sql.Tx) error {
 	_, err := tx.Exec(`
-CREATE TABLE windows_mdm_pending_commands (
+CREATE TABLE old_windows_mdm_pending_commands (
```

Comment:
My goal is to just delete this DB migration once this merges. I assume this is OK, the `feat_windows_msmdm` branch has never been deployed anywhere and it will not cause DB migrations issues?

Reply:
Yes, `feat_windows_msmdm` has never been deployed so it should be fine